### PR TITLE
Stop using stlport in Android.

### PR DESCRIPTION
**Pull request by DanAlbert**
Stlport has been deprecated in Android for a while now, and is finally
being removed very soon.

### CODE REVIEW

File: parameter/Android.mk [hunk 1]
```diff
@@ -125,7 +125,8 @@ common_cflags := \
         -Wall \
         -Werror \
         -Wextra \
-        -Wno-unused-parameter
+        -Wno-unused-parameter \
+        -Wno-maybe-uninitialized \
```
Comment (krocard) [inline parameter/Android.mk]: Why is this needed ? Is it linked to the new lib std  ?

**Comment by krocard** [comment]
We will merge that after the next android release as this patch on android-L breaks the compilation (no default stl).

**Comment by dawagner** [comment]
Done in tag v2.6.0+no-stlport.
